tests/unit/test_estimator.py

```diff
@@ -325,15 +325,18 @@ def test_init_with_source_dir_s3(strftime, sagemaker_session):
 
 
 def test_framework_name_from_framework_image():
-    framework, py_ver = framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-mxnet-py2-gpu:1')
+    image_name = '123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-mxnet-py2-gpu:2.5.6-gpu-py2'
+    framework, py_ver, tag = framework_name_from_image(image_name)
     assert framework == 'mxnet'
     assert py_ver == 'py2'
+    assert tag == '2.5.6-gpu-py2'
 
 
 def test_framework_name_from_other():
-    framework, py_ver = framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-myown-py2-gpu:1')
+    framework, py_ver, tag = framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-myown-py2-gpu:1')
     assert framework is None
     assert py_ver is None
+    assert tag is None
```

Comment:
Why would we get 'None' here? The tag will be '1', right?

Reply:
Not the way framework_name_from_image works. In this case it:
```
        name_pattern = re.compile('^sagemaker-(tensorflow|mxnet)-(py2|py3)-(cpu|gpu):(.*)$')
        name_match = name_pattern.match(sagemaker_match.group(8))
```
name_match is None and everything is None as a result.
I don't see the value in changing this behavior since we can't really use the tag if we don't have the framework, similar to python version.

Reply:
Ah yes, sorry - I missed the 'myown' framework name.